server/src/main/java/org/opensearch/action/search/SearchResponse.java

```diff
@@ -604,6 +658,76 @@ public String toString() {
         }
     }
 
+    /**
+     * Holds info about the clusters that the search was executed on: how many in total, how many of them were successful
+     * and how many of them were skipped.
+     *
+     * @opensearch.internal
+     */
+    public static class PhaseTook implements ToXContentFragment, Writeable {
+        public static final PhaseTook EMPTY = new PhaseTook();
+
+        static final ParseField PHASE_TOOK = new ParseField("phase_took");
+        private final Map<String, Long> phaseStatsMap;
+
+        // Private constructor for empty object
+        private PhaseTook() {
+            Map<String, Long> defaultPhaseTookMap = new HashMap<>();
+            for (SearchPhaseName searchPhaseName : SearchPhaseName.values()) {
+                defaultPhaseTookMap.put(searchPhaseName.getName(), (long) -1);
+            }
+            this.phaseStatsMap = defaultPhaseTookMap;
+        }
```

Comment:
Usage of defaultPhaseTookMap isn't really required.

Reply:
Plus why do you even need to initialize it with -1? Just keep an empty map with no values.
If took time is not enabled, don't do anything, and using empty map we can know that we don't need to put anything in response.

Using -1 is just complicating this.

Reply:
removed